src/main/java/org/carlmontrobotics/subsystems/Arm.java

```diff
@@ -1,48 +1,209 @@
+// Wrist angle is measured relative to the arm with 0 being parallel to the arm and bounded between -π and π (Center of Mass of Roller)
 public class Arm extends SubsystemBase {
+  ///GOALS
+  /*
+   * public static final double intakeAngle = Math.toRadians(0);
+		public static final double ampAngle = Math.toRadians(103);
+		public static final double placeholderSpeakerAngle1 = Math.toRadians(24);
+		public static final double placeholderSpeakerAngle2 = Math.toRadians(24);
+		public static final double placeholderSpeakerAngle3 = Math.toRadians(24);
+		public static final double climberUpAngle = Math.toRadians(24);
+		public static final double climberDownAngle = Math.toRadians(24);
+   */
+    // a boolean meant to tell if the arm is in a forbidden posistion AKA FORBIDDEN FLAG
+    
+    private final CANSparkMax armMotor1 = MotorControllerFactory.createSparkMax(ARM_MOTOR_PORT_1, MotorConfig.NEO);
+    private final CANSparkMax armMotor2 = MotorControllerFactory.createSparkMax(ARM_MOTOR_PORT_2, MotorConfig.NEO);
+    private final RelativeEncoder armEncoder = armMotor1.getEncoder();
+
+
+    private static double kDt = 0.02;
+   
+    //PID, feedforward, trap profile
+    private final SimpleMotorFeedforward armFeed = new SimpleMotorFeedforward(kS, kV, kA);
+    private final SparkPIDController armPID1 = armMotor1.getPIDController();
+    private final SparkPIDController armPID2 = armMotor2.getPIDController();
+    private TrapezoidProfile armProfile = new TrapezoidProfile(trapConstraints);
+    private Timer armProfileTimer = new Timer();
+    TrapezoidProfile.State goalState = new TrapezoidProfile.State(0,0);//TODO: update pos later
+
+    // rad, rad/s
+    //public static TrapezoidProfile.State[] goalState = { new TrapezoidProfile.State(-Math.PI / 2, 0), new TrapezoidProfile.State(0, 0) };
+
     public Arm() {
-			//arm 
-      /*
-       have 3 set positions
-       Speaker position 
-       Amp position
-        ground position
-       These set positions would also be conrtolled by buttons
-
-       There will also be a manual control for the arm using the right joystick
-      */
-    }
-	
-    public void setArmPos() {
-      //Sets arm to the optimal angle for amp, speaker and Ground intake | used to score in amp
-      //these values are in constants
-      //pass in where scorign and use switch statement
+      // weird math stuff
+        armMotor1.setInverted(motorInverted);
+        armMotor1.setIdleMode(IdleMode.kBrake);
+        armMotor2.setInverted(motorInverted);
+        armMotor2.setIdleMode(IdleMode.kBrake);
+        armEncoder.setPositionConversionFactor(rotationToRad);
+        armEncoder.setVelocityConversionFactor(rotationToRad);
+        armEncoder.setInverted(encoderInverted);
+
+        armMotor2.follow(armMotor1);
+     
+        //armEncoder1.setZeroOffset(offsetRad);
+      
+        //armPID.setTolerance(posToleranceRad, velToleranceRadPSec);
+
```

Comment:
You need to make pid use the correct encoder. It should not be using the default encoder. In addition, since you are using an absolute encoder, the encoder will not read values past a full rotation, so you need to do some wrapping.